src/gui/qgsrelationeditorwidget.cpp

```diff
@@ -198,10 +259,19 @@ void QgsRelationEditorWidget::setRelationFeature( const QgsRelation &relation, c
   if ( mVisible )
   {
     QgsFeatureRequest myRequest = mRelation.getRelatedFeaturesRequest( mFeature );
-    mDualView->init( mRelation.referencingLayer(), nullptr, myRequest, mEditorContext );
+    initDualView( mRelation.referencingLayer(), myRequest );
   }
 }
 
+void QgsRelationEditorWidget::initDualView( QgsVectorLayer *layer, const QgsFeatureRequest &request )
+{
+  mDualView->init( layer, nullptr, request, mEditorContext );
+  mFeatureSelectionMgr = new QgsFilteredSelectionManager( layer, request, mDualView );
+  mDualView->setFeatureSelectionManager( mFeatureSelectionMgr );
```

Comment:
@troopa81 
I am getting crashes from here
![image](https://user-images.githubusercontent.com/127259/69792347-be2c6a00-11c6-11ea-947a-2993cf80ef1b.png)

Reply:
@3nids Doing what exactly? Did you create an issue? Sorry for late answer, I was on leave